commands/nic.go

```diff
@@ -233,14 +234,20 @@ func RunNicGet(c *core.CommandConfig) error {
 }
 
 func RunNicCreate(c *core.CommandConfig) error {
-	nic, resp, err := c.Nics().Create(
-		viper.GetString(core.GetGlobalFlagName(c.Resource, config.ArgDataCenterId)),
-		viper.GetString(core.GetGlobalFlagName(c.Resource, config.ArgServerId)),
-		viper.GetString(core.GetFlagName(c.NS, config.ArgName)),
-		viper.GetStringSlice(core.GetFlagName(c.NS, config.ArgIps)),
-		viper.GetBool(core.GetFlagName(c.NS, config.ArgDhcp)),
-		viper.GetInt32(core.GetFlagName(c.NS, config.ArgLanId)),
-	)
+	dcId := viper.GetString(core.GetGlobalFlagName(c.Resource, config.ArgDataCenterId))
+	serverId := viper.GetString(core.GetGlobalFlagName(c.Resource, config.ArgServerId))
+	name := viper.GetString(core.GetFlagName(c.NS, config.ArgName))
+	ips := viper.GetStringSlice(core.GetFlagName(c.NS, config.ArgIps))
+	dhcp := viper.GetBool(core.GetFlagName(c.NS, config.ArgDhcp))
+	lanId := viper.GetInt32(core.GetFlagName(c.NS, config.ArgLanId))
+
+	c.Printer.Infof("Properties set for creating the nic: DataCenterId: %v, ServerId: %v, Name: %v, Ips: %v, Dhcp: %v, Lan: %v",
```

Comment:
I personally think DatacenterId is not a property for NIC and it should be outputed in a different message before properties. Same for ServerId.

Reply:
done